pyproject.toml

```diff
@@ -1,56 +1,93 @@
-[tool.poetry]
+[build-system]
+requires = [
+    "hatchling>=1.13.0",
+]
+build-backend = "hatchling.build"
+
+[project]
 name = "servicex"
 version = "3.0.0-alpha.2"
-description = ""
+description = "Python SDK and CLI Client for ServiceX"
+readme = "README.md"
+license = { text = "BSD-3-Clause" }  # SPDX short identifier
+requires-python = ">=3.8"
 authors = [
-    "Ben Galewsky <bengal1@illinois.edu>",
-    "Gordon Watts <gwatts@uw.edu>",
+    { name = "Ben Galewsky", email = "bengal1@illinois.edu" },
+    { name = "Gordon Watts", email = "gwatts@uw.edu" },
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+dependencies = [
+    "qastle>=0.17",
+    "func_adl>=3.2.6",
+    "requests>=2.31",
+    "pydantic>=1.10,<2.0",  # FIXME: Adopt pydantic 2.0 API
+    "httpx>=0.24",
+    "miniopy-async>=1.15",
+    "tinydb>=4.7",
+    "google-auth>=2.17",
+    "typer[all]>=0.9.0",
+    "PyYAML>=6.0",
+    "types-PyYAML>=6.0",
+    "rich>=13.0.0",  # databinder
+    "nest-asyncio>=1.5.7",  # databinder # FIXME: nest-asyncio deprecated
```

Comment:
These dependencies are required by `servicex.databinder` but were not specified.